_episodes/08-crop-raster-data.md

```diff
@@ -294,66 +234,87 @@ wells.plot(ax=ax, color='red', markersize=2)
+> {: .solution}
+{: .challenge}
 
-> ## Exercise: Select the raster data around the dike
-> The dikes are stored as polylines in `.shp` file in `data/dikes`. Let's select out all the raster data within 100m around the dikes and visualize the results.
+> ## Exercise: Select the raster data around the waterways
+> In the previous episode we have corrected the waterway vector data and saved it in `waterways_nl_corrected.shp`. Can you select out all the raster data within 100m around the waterways, and visualize the results?
 >
 > > ## Solution
 > > ~~~
-> > # Load dikes polyline
-> > dikes = gpd.read_file("data/dikes/dikes.shp")
-> > dikes = dikes.to_crs(true_color_image.rio.crs)
-> > # Dike buffer
-> > dikes_buffer = dikes.buffer(100)
+> > # Load waterways polyline and convert CRS
+> > waterways_nl = gpd.read_file("waterways_nl_corrected.shp")
+> > waterways_nl = waterways_nl.to_crs(raster_clip.rio.crs)
+> > 
+> > # Crop the waterways to the image extent
+> > xmin, xmax = raster_clip.x[[0, -1]]
+> > ymin, ymax = raster_clip.y[[0, -1]]
+> > waterways_nl = waterways_nl.cx[xmin:xmax, ymin:ymax]
+> > 
+> > # waterways buffer
+> > waterways_nl_buffer = waterways_nl.buffer(100)
+> > 
 > > # Crop
-> > raster_clip_dikes = raster_clip.rio.clip(dikes_buffer, dikes_buffer.crs)
+> > raster_clip_waterways = raster_clip.rio.clip(waterways_nl_buffer, waterways_nl_buffer.crs)
+> > 
 > > # Visualize
-> > raster_clip_dikes.plot.imshow(figsize=(8,8))
+> > raster_clip_waterways.plot.imshow(figsize=(8,8))
 > > ~~~
 > > {: .language-python}
-> > <img src="../fig/20-crop-raster-dike-solution-07.png" title="Raster croped by buffer around dikes" width="512" style="display: block; margin: auto;" />
+> > <img src="../fig/20-crop-waterways-07.png" title="Raster croped by buffer around waterways" width="512" style="display: block; margin: auto;" />
 > {: .solution}
 {: .challenge}
 
 ## Crop raster data using another raster data
 
-Imagine that we have two raster datasets, let's say the `true_color_image` and `crop_fields` in
-different coordinate systems. Our goal is to crop the `true_color_image` image using the
+So far we have learnt how to crop raster image with vector data. We can also crop a raster with another raster data. In this section, we will demonstrate how to crop the `true_color_image` image using the
 `crop_fields` image.
 
 > ## Using `crop_fields` raster image
 >
-> For this section, we will use the `crop_fields.tif` image that was produced in
-> the exercise "**Select the raster data within crop fields**".
+> For this section, we will use the `crop_fields.tif` image that was produced in the section "**Crop raster data with polygon**".
 {: .callout}
 
-We read in both images and check their coordinate system:
+We read in the `crop_fields.tif` image and reproject it to the RD CRS system:
 ~~~
 # Read crop_fields
 crop_fields = rioxarray.open_rasterio("crop_fields.tif")
+
+# Reproject to RD
+crop_fields = crop_fields.rio.reproject("EPSG:28992")
```

Comment:
I am not sure why we need to reproject the image here? When doing `retroject_match` we would anyway convert `true_color_image` to the CRS of `crop_fields`, whatever that is - right?

Reply:
Yes. Actually this is an artificial step to put them in different CRS. The purpose of this is to show that the `retroject_match` can do reprojection and match in one step. Will add some more description text for this.